### ISSUE TITLE
Deno bundle not bundling custom Errors

### ISSUE THREAD

**Issue by DanielRamosAcosta** [denoland/deno#9176]
When trying to bundle a code that have imports of custom Errors that are only used to `throw` them won't be bundled.

Here is a gist that reproduces the error:  https://gist.github.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee

```
$ deno --version
deno 1.6.3 (release, x86_64-pc-windows-msvc)
v8 8.8.294
typescript 4.1.3

$ deno bundle https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/mod.ts
Bundle https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/mod.ts
Check https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/mod.ts
function example2() {
    throw new MyError();
}
const example1 = example2;
export { example1 as example };
```

But when bundleing the `only-trow.ts`  file directly, it does it ok:

```
$ deno bundle https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/only-throw.ts
Bundle https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/only-throw.ts
Check https://gist.githubusercontent.com/DanielRamosAcosta/9044f1dc100991be595e09980dea93ee/raw/d4c71ec94e174a4d48c486d6d01cc80f20034bcb/only-throw.ts
class MyError extends Error {
    constructor(){
        super("I'm in?");
    }
}
export function example() {
    throw new MyError();
}
```

**Comment by kitsonk** [comment]
cc/ @kdy1

**Comment by kdy1** [comment]
It seems like it's already fixed in the upstream.

I get

```ts
class MyError extends Error {
    constructor(){
        super("I'm in?");
    }
}
const MyError1 = MyError;
const MyError2 = MyError1;
function example() {
    throw new MyError2();
}
const example1 = example;
export { example as example };

```

from the bundler.


Note that after https://github.com/swc-project/swc/pull/1342 is merged, it will be


```
class MyError extends Error {
    constructor(){
        super("I'm in?");
    }
}
const MyError1 = MyError;
const MyError2 = MyError1;
function example1() {
    throw new MyError2();
}
export { example1 as example };

```

and with inlining enabled, it will be

```ts
class MyError extends Error {
    constructor(){
        super("I'm in?");
    }
}
function example1() {
    throw new MyError();
}
export { example1 as example };

```

**Comment by kitsonk** [comment]
Ok we will close when we update.

**Comment by DanielRamosAcosta** [comment]
This is fixed in Deno 1.7.0 @kitsonk, I think this could be closed! 😄
